leveldbjni/src/main/native-package/m4/ltoptions.m4

```diff
@@ -1,14 +1,14 @@
 # Helper functions for option handling.                    -*- Autoconf -*-
 #
-#   Copyright (C) 2004, 2005, 2007, 2008, 2009 Free Software Foundation,
```

Comment:
Why changes to the comment header? Are those changes automatically generated somehow? If so, can you not include those automatically generated files in your pull request?

Reply:
Yes, they are auto-generated. 
https://github.com/utsaslab/leveldbjni/pull/1/files/4ece7fed84bed249e85ecc72d6f4e81d68a883b9#r219715836